### PR TITLE
96%

### PR DESCRIPTION
**Pull request by Unnunu**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 9 functions (+0.12%) on `us`.

**Review by ethteck — APPROVED** [review]

File: include/macros.h [hunk 1]
```diff
@@ -26,6 +26,13 @@
 #define NOP_FIX __asm__(".set nogpopt");
 #define NOP_UNFIX __asm__(".set gpopt");
 
+#define SWAP16(in, out) \
```
Comment (nanaian) [inline include/macros.h:29]: Unsure what I think of this given we only use it once in this PR but perhaps I'll change my mind if it sees use elsewhere

**Review by nanaian — APPROVED** [review]

File: include/macros.h (hunk 1 above)
Comment (HailSanta) [reply include/macros.h:29]: I think it should go in the file its associated with, that's how its arranged in naudio.

Comment (ethteck) [reply include/macros.h:29]: Sounds grood

**Pull request merged**